### PR TITLE
Feature/fe new project form

### PR DESCRIPTION
**Pull request by fluffysheepcloud**
To see how it looks like use /NewProjectForm. Connected to backend somewhat

### CODE REVIEW

File: frontend/src/pages/NewProjectForm.jsx [hunk 1]
```diff
@@ -1,244 +1,129 @@
...
-              <Paper elevation = {0} className={classes.paper}> 
+            <Grid item xs = {12} >
+                <TextField label = "Tell us about your project! " rows = "10"  multiline = {true}/>
+            </Grid>
+           
+            <Grid item xs = {12} >
+             <TextField label = "Qualifications for potential team members" rows = "3" multiline = {true}/>
+            </Grid>
+            <Grid  item  xs = {3} >
+                Start Date*
```
Comment (PeterYangIO) [inline frontend/src/pages/NewProjectForm.jsx:72]: The `required` property on textfields automatically adds the asterisk for you. They also provide a consistent UI experience for providing the name of the field through the `label` property

https://material-ui.com/components/text-fields/

File: frontend/src/components/SideBarOrganization.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import ListItemText from '@material-ui/core/ListItemText';
+import AccountBoxIcon from '@material-ui/icons/AccountBox';
+import FolderIcon from '@material-ui/icons/Folder';
+import CreateIcon from '@material-ui/icons/Create';
+import GroupIcon from '@material-ui/icons/Group';
+import PersonIcon from '@material-ui/icons/Person';
+
+//use styling hook here to align name and png next to each other
+//change links for after user type is merged so you dont have a merge conflict with app js
+const sideBarStyles = makeStyles((theme)  => ({
```
Comment (PeterYangIO) [inline frontend/src/components/SideBarOrganization.jsx:18]: Is it intentional that the two sidebars have different styles?

File: frontend/src/pages/NewProjectForm.jsx (hunk 1 above)
Comment (fluffysheepcloud) [reply frontend/src/pages/NewProjectForm.jsx:72]: I will look into that one that sounds better

File: frontend/src/components/SideBarOrganization.jsx (hunk 2 above)
Comment (fluffysheepcloud) [reply frontend/src/components/SideBarOrganization.jsx:18]: Actually this sidebar is outdated. I will update it in my next commit

**Comment by anh65498** [comment]
Please add top-padding and bottom-padding to the white container of the page. Make "Tell us about yourself" and "Qualification" text box the same height

**Comment by anh65498** [comment]
I have merged branch "Feature/fe your projects: Backend Connection" into "develop", which contains <NewProjectForm>

**Pull request closed without merging**